prac_06/REFLECTION.md

```diff
@@ -5,21 +5,21 @@ Replace each `...` with your meaningful answer.
 
 ## What have you changed about your process in this subject based on your personal reflection?
 
-...
+I've started recording time it takes for me to start and finish.
 
 ## What do you still need to change in terms of your approach, application, time on task, attitude, etc.?
 
-...
+I have the bad habit of being completely distracted, such as cooking, playing games, etc while coding since it's a way for me to relief stress. 
 
 ## What are you doing well in the subject so far that you will keep doing?
 
-...
+I've gotten a lot more focused and I've kept up with the accessments instead of fixing 
 
 ## What do you need to stop or change in your work for this subject overall so far?
 
-...
+I need to give myself time to breathe with Pseudocode because I am very impatient and I like to get started instantly. This also causes problems later on while coding because I tend to have the plans in my head, which I'm too lazy to write down. This also result in some parts of the code being rewritten or discarded entirely when I write it again on a different time or day.
 
 ## What is one request you have or change you suggest for this subject and/or your IT degree overall?
 
-...
+I don't have any suggestion.
 
```

Comment:
I can not find the source code.  Please reference to : https://github.com/CP1404/Practicals/tree/master/prac_06  Deliverables. You should submit the files. such as *.md and *.py etc.